### PR TITLE
Translate Adafruit workshop to Brazilian Portuguese

### PR DESCRIPTION
**Pull request by aminueza**
This PR changes:

- Translate Adafruit workshop to Brazilian Portuguese

### CODE REVIEW

**Review by felladrin — APPROVED** [review]
Excellent Brazilian Portuguese translation!

**Review by ozhang22 — COMMENTED** [review]
I will need to do some additional changes first to allow the portuguese language option to show up on the website - please stand by

**Comment by ozhang22** [comment]
@aminueza Can you please pull the latest `master` branch? There are two additional files that will need a quick translation: `content\brazilian-portuguese\index.md` and `themes\docdock\i18n\pt-br.toml`.

**Comment by aminueza** [comment]
> @aminueza Can you please pull the latest `master` branch? There are two additional files that will need a quick translation: `content\brazilian-portuguese\index.md` and `themes\docdock\i18n\pt-br.toml`.

I have just translated those tho files. Plus, I changed some words for a better understanding of Portuguese.

File: content/brazilian-portuguese/_index.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
-## Workshop Projects
+## Projetos do Minicurso
+
 {{< workshop-list style="card" depth="1" showhidden="false" description="true"  />}}
 
-## Contributing
-Want to help improve our workshops or our website? Click here: **[Workshop Contributor Guidelines](guidelines/)**!
+## Colabore conosco
+
+Gostaria de nos ajudar a melhorar nossos minicursos ou nosso site? Clique aqui: **[Guida de contribuição para minicursos](guidelines/)**!
```
Comment (felladrin) [inline content/brazilian-portuguese/_index.md:14]:
```suggestion
Gostaria de nos ajudar a melhorar nossos minicursos ou nosso site? Clique aqui: **[Guia de contribuição para minicursos](guidelines/)**!
```

**Review by ozhang22 — APPROVED** [review]

**Pull request merged**